Azure/azure-key-vault/ingest/parser.yml

```
@@ -0,0 +1,33 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: "database"
+          event.dataset: "keyvault"
+          event.outcome: "{{json_event.message.resultSignature}}"
+          event.action: "{{json_event.message.operationName}}"
+          event.type: ["access", "change", "info"]
+          cloud.provider: "azure"
+          "@timestamp": "{{json_event.message.time}}"
+          source.ip: "{{json_event.message.callerIpAddress}}"
+          user.name: "{{json_event.message.identity.claim.unique_name}}"
+          user_agent.original: "{{json_event.message.properties.clientInfo}}"
+          url.original: "{{json_event.message.properties.requestUri}}"
+          tls.version: "{{json_event.message.properties.tlsVersion}}"
+          azure.key_vault.result.type: "{{json_event.message.resultType}}"
+          azure.key_vault.result.description: "{{json_event.message.resultDescription}}"
+          azure.key_vault.result.signature: "{{json_event.message.resultSignature}}"
+          azure.key_vault.correlation_id: "{{json_event.message.correlationId}}"
+          azure.key_vault.resource_id: "{{json_event.message.resourceId}}"
+
+      - set:
+          event.type: ["denied", "change", "info"]
```

`denied` doesn't match the `database` category: https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html#ecs-event-category-database
I suggest using only one type:
```suggestion
          event.type: ["access"]
```